src/mixins/enum-mixin.ts

```diff
@@ -176,6 +176,11 @@ export default class EnumMixin extends Vue {
     return (item.name === FilingTypes.SPECIAL_RESOLUTION)
   }
 
+  /** Returns True if filing is a Limited Restoration. */
+  isTypeLimitedRestoration (item: any): boolean {
+    return (item.name === FilingTypes.RESTORATION && item.data?.restoration?.type === 'limitedRestoration')
```

Comment:
I agree with the optional chaining here, since this might not be a restoration item.

You should use an enum for the type -- see RestorationTypes.

Reply:
Good catch.  I've fixed it.